IngresseSDK/Model/Requests/EventRequests.swift

```diff
@@ -28,5 +28,16 @@ extension Request {
 
             public init() {}
         }
+        
+        public struct Search {
+            public var filters: [String: String] = [:]
+            public var orderBy: String = ""
+            public var from: String = ""
+            public var to: String = ""
+            public var offset: Int = 0
+            public var size: Int = 0
+            
+            public init() {}
+        }
```

Comment:
## Properties

1. Buscar sempre manter as properties como imutáveis `let`;
1. Não deixar valores **defaults** se não forem extremamente necessários;
1. Passar os valores no construtor como obrigatórios. Isso garante que quem for usar essa **struct** não esqueça de nenhum valor.

## Encodable

Pensei em usar o swift a nosso favor. Então usar o protocolo `Encodable` para permitir de maneira fácil a gente adicionar parâmetros na `URLBuilder`.

```suggestion
        public struct Search: Encodable {

            public let orderBy: String
            public let from: String
            public let to: String
            public let offset: Int
            public let size: Int
            public let title: String
            public let description: String

            public init(orderBy: String,
                        from: String,
                        to: String,
                        offset: Int,
                        size: Int,
                        title: String,
                        description: String) {

                self.orderBy = orderBy
                self.from = from
                self.to = to
                self.offset = offset
                self.size = size
                self.title = title
                self.description = description
            }
        }
```